src/diffusers/models/modeling_utils.py

```diff
@@ -721,10 +805,6 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             token=token,
             revision=revision,
             subfolder=subfolder,
-            device_map=device_map,
-            max_memory=max_memory,
-            offload_folder=offload_folder,
-            offload_state_dict=offload_state_dict,
```

Comment:
I am not sure why we used to pass this but these are NOT used in `configuration_utils.py` anywhere. Given that, I think they are best removed:

* No unwanted cognitive burden in thinking about what these are doing for configuration parsing. 
* Reduces LoC (albeit small)